demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -139,6 +139,142 @@ class DemoTileOrganizer {
   }
 }
 
+class DemoVideoTile extends HTMLElement {
+  public set tileIndex(tileIndex: number) {
+    // Update IDs for integration tests which need them
+    this.querySelector('.video-tile-nameplate').id = `nameplate-${tileIndex}`
+    this.querySelector('.video-tile-video').id = `video-${tileIndex}`
+  }
+
+  public set showConfigDropdown(shouldShow: boolean) {
+    const display: string = shouldShow ? 'block' : 'none';
+    (this.querySelector('.button-video-tile-config-drop') as HTMLElement).style.display = display;
+    (this.querySelector('.video-tile-pause') as HTMLElement).style.display = display;
+  }
+
+  public set showRemoteVideoPreferences(shouldShow: boolean) {
+    const display: string = shouldShow? 'block' : 'none';
+    (this.querySelector('.target-resolution-header') as HTMLElement).style.display = display;
+    (this.querySelector('.target-resolution-toggle') as HTMLElement).style.display = display;
+    (this.querySelector('.video-priority-header') as HTMLElement).style.display = display;
+    (this.querySelector('.video-priority-toggle') as HTMLElement).style.display = display;
+  }
+
+  show(isContent: boolean) {
+    this.classList.add(`active`);
+    if (isContent) {
+      this.classList.add('content');
+    }
+  }
+
+  public set featured(featured: boolean) {
+    if (featured) {
+      this.classList.add(`featured`);
+    } else {
+      this.classList.remove(`featured`);
+    }
+  }
```

Comment:
Standardize on straight single quotes for strings that don't actually use templating.